pandas/core/groupby.py

```diff
@@ -1476,8 +1476,93 @@ def describe(self, **kwargs):
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
         """
-        Provide resampling when using a TimeGrouper
-        Return a new grouper with our resampler appended
+        Provide resampling when using a TimeGrouper.
+
+        Given a grouper the function resamples it according to a string and an
```

Comment:
"string" -> "frequency" and link to the frequency aliases:

    :ref:`frequency aliases <timeseries.offset_aliases>`